src/main/views/crown-daily-list.njk

```diff
@@ -0,0 +1,210 @@
+{% from "govuk/components/table/macro.njk" import govukTable as 'govukTable' %}
+{% from "./macros/common-components.njk" import goBack %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+{% extends "template.njk" %}
+{% block pageTitle %}
+  {{ title }}
+{% endblock %}
+{% block beforeContent %}
+  {{ super() }}
+  {{ goBack(text = backButton) }}
+{% endblock %}
+{% block content %}
+  <div class="parent-box">
+    <h2 id='page-heading' class="govuk-heading-l">
+      {{ courtName }}: {{ headerPart1 }}
+      {% if bill %}
+        <br />/{{ courtName }}: {{ headerPart2 }}
+      {% endif %}
+    </h2>
+    <p class="govuk-body govuk-!-font-weight-bold">{{ listDate }} {{ contentDate }}</p>
+    <p class="govuk-body">{{ listUpdated | replace("DATE", publishedDate) }} {{ publishedTime }}</p>
+    <p class="govuk-body">{{ draft }} {{ versionText }} {{ version }}</p><br/>
+    <p class="govuk-body">
+      {% for line in listData['venue']['venueAddress']['line'] %}
```

Comment:
Also need to include town / county here if present

Reply:
done.